internal/pkg/manifest/manifest.go

```diff
@@ -94,6 +94,11 @@ func LoadManifest(fs filesystem.Fs) (*Manifest, error) {
 		}
 	}
 
+	// Validate
```

Comment:
Validacia `manifest.json` presunuta pred `Connect records together and resolve parent path`.
Ak je manifest nevalidny, nema zmysel pokracovat -> a chybove hlasky su viac zrozumitelne, ako keby to spadlo az dalej.